nss-day-cohort-48/giffygram-archies-bunker2-0
# pseudo code and profile page with following

#### Changes Made

1. Modified file `everything` to include `pseudo code` and functionality.
2. created the `Profile.js` to include the profile page
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout thejmdw-profile
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    >
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
